tark/tark_web/views.py

```diff
@@ -445,3 +445,89 @@ def transcript_details(request, stable_id_with_version, search_identifier):
                                                                'transcript_history': transcript_history,
                                                                'search_identifier': search_identifier,
                                                                'stable_id_with_version': stable_id_with_version})
+
+# queryfor manelist
+def mane_list_new(request):
+    sql = """
+        SELECT DISTINCT
+                        t1.transcript_id, t1.stable_id as ens_stable_id, t1.stable_id_version as ens_stable_id_version,
+                        relationship_type.shortname as mane_type,
+                        t2.stable_id as refseq_stable_id, t2.stable_id_version as refseq_stable_id_version,
+                        gn1.name as ens_gene_name
+                        FROM
+                        transcript t1
+                        JOIN transcript_release_tag trt1 ON t1.transcript_id=trt1.feature_id
+                        JOIN transcript_release_tag_relationship ON
+                        trt1.transcript_release_id=transcript_release_tag_relationship.transcript_release_object_id
+                        JOIN transcript_release_tag trt2 ON
+                        transcript_release_tag_relationship.transcript_release_subject_id=trt2.transcript_release_id
+                        JOIN transcript t2 ON trt2.feature_id=t2.transcript_id
+                        JOIN relationship_type ON
+                        transcript_release_tag_relationship.relationship_type_id=relationship_type.relationship_type_id
+                         JOIN transcript_gene tg1 ON
+                        t1.transcript_id=tg1.transcript_id
+                        JOIN gene gene1 ON
+                        tg1.gene_id=gene1.gene_id
+                        JOIN gene_names gn1 ON
+                        gene1.name_id=gn1.external_id
+                        where gn1.primary_id=1;
+    """
+    # print(sql)
```

Comment:
this print can be deleted

Reply:
Thanks @bilalebi , this has now been removed